core/app/[locale]/error.tsx

```diff
@@ -1,5 +1,16 @@
 'use client';
 
-import { lazy } from 'react';
+import { useTranslations } from 'next-intl';
 
-export default lazy(() => import('./_components/error'));
```

Comment:
We were doing this to prevent `next-intl` from loading as part of the initial bundle, but at this point I don't know if this is happening anyway since we use the context. More info [here](https://next-intl.dev/docs/environments/error-files).